lib/app.jsx

```diff
@@ -507,6 +508,18 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 					</ContextMenu>
 				}
 				{ isAuthorized ?
+					<div className="simplenote-app">
+						<TopBar
+							note={selectedNote}
+							onNewNote={ this.onNewNote }
+							onTrashNote={this.onTrashNote}
+							onRestoreNote={this.onRestoreNote}
+							onShareNote={this.onShareNote}
+							onDeleteNoteForever={this.onDeleteNoteForever}
+							onRevisions={this.onRevisions}
+							onCloseNote={ () => this.props.actions.closeNote() }
+							onNoteInfo={ () => this.props.actions.toggleNoteInfo() }
```

Comment:
Oh damn, I see what you meant now!
Yeah, that's a byproduct of me blindly copying the props from `<NoteEditor>` a bunch of lines below, where it creates those two closures.